workos/resources/audit_logs.py

```diff
@@ -1,7 +1,8 @@
-from typing import Literal, Optional
+from typing import Any, Dict, Literal, Optional
 from workos.resources.workos_model import WorkOSModel
 
 AuditLogExportState = Literal["error", "pending", "ready"]
+AuditLogMetadata = Dict[str, Any]
```

Comment:
Should this be `Mapping`?
```suggestion
AuditLogMetadata = Mapping[str, Any]
```